caffe2/core/nomnigraph/include/nomnigraph/Graph/Graph.h

```diff
@@ -117,6 +117,14 @@ class Node : public StorageType<T>, public Notifier<Node<T, U>> {
     return inEdges;
   }
 
+  void setInEdges(std::vector<EdgeRef> es) {
```

Comment:
General question, why don't we use `const std::vector<EdgeRef>&` as input signature? Won't this incur any extra copy if input is not rvalue?

Reply:
I think it'll be optimized away, but I felt the signature should reveal that a copy operation is happening (as it is necessary)